### PR TITLE
Clean command

### PR DESCRIPTION
**Pull request by enchev**
Implemented #1580

### CODE REVIEW

File: docs/man_pages/project/configuration/platform-clean.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+platform remove
```
Comment (PanayotCankov) [inline docs/man_pages/project/configuration/platform-clean.md]: clean

File: lib/commands/platform-clean.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+export class CleanCommand implements ICommand {
+	constructor(private $platformService: IPlatformService,
+		private $errors: IErrors) { }
+
+	public execute(args: string[]): IFuture<void> {
+		return (() => {
+			this.$platformService.removePlatforms(args).wait();
```
Comment (PanayotCankov) [inline lib/commands/platform-clean.ts]: One last test would be to mocks the "removePlatform" and "addPlatform" and asserts they were executed, and were executed in that order, and the platforms passed in the args of the clean command matches the platforms passed to removePlatforms/addPlatforms methods.

**Comment by PanayotCankov** [comment]
:+1:

**Comment by PanayotCankov** [comment]
If the `tns clean ios|android` won't preserve an installed custom version of the ios/android runtimes, maybe it is worth to mention that in the command documentation.

**Comment by enchev** [comment]
run ci

**Comment by enchev** [comment]
run ci

**Pull request merged**